schema/partner.js

```diff
@@ -70,6 +70,10 @@ const PartnerType = new GraphQLObjectType({
         type: GraphQLBoolean,
         resolve: ({ pre_qualify }) => pre_qualify,
       },
+      is_fair_partner: {
+        type: GraphQLBoolean,
+        resolve: ({ claimed }) => !claimed,
```

Comment:
Yeah, I think the only reason for keeping `claimed` around would be not migrating any kind of reports done on partner (cc @starsirius ). I'll take a look around and see if this is actually the case. Hold this PR!